generators/app/templates/my-awesome-site/gulpfile.babel

```diff
@@ -74,17 +74,19 @@ gulp.task('css', () => {
 <% if (includeSass || includeScss) { -%>
 // Compile scss to css.
 gulp.task('scss', () => {
+//This will check for sass in all pages of the site, incase you use them
```

Comment:
Please add a space after the `//`, before starting the comment.
And add `scss` in the comment,  like `sass` => `sass/scss`